db-migrations/5_legal_basis.up.sql

```diff
@@ -8,6 +8,7 @@ INSERT INTO survey.legalbasis ( ref, longname ) VALUES ( 'STA1947', 'Statistics
 INSERT INTO survey.legalbasis ( ref, longname ) VALUES ( 'STA1947_BEIS', 'Statistics of Trade Act 1947 - BEIS' );
 INSERT INTO survey.legalbasis ( ref, longname ) VALUES ( 'Vol', 'Voluntary Not Stated' );
 INSERT INTO survey.legalbasis ( ref, longname ) VALUES ( 'Vol_BEIS', 'Voluntary - BEIS' );
+INSERT INTO survey.legalbasis ( ref, longname ) VALUES ( 'Voluntary', 'Voluntary' );
```

Comment:
not entirely sure how migrations are in golang but wont this require a new migration script? Presumably script 1 - 5 have already been run in production.